enhancements/installer/cloud-custom-dns.md

```diff
@@ -0,0 +1,623 @@
+---
+title: cloud-custom-dns
+authors:
+  - "@sadasu"
+reviewers: 
+  - "@patrickdillon, for Installer aspects"
+  - "@Miciah, for Ingress Controller aspects"
+  - "@cybertron, for on-prem networking aspects"
+  - "@2uasimojo, for hive expertise"
+approvers:
```

Comment:
From the [template](https://github.com/openshift/enhancements/blob/master/guidelines/enhancement_template.md?plain=1#L7):

> A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.

(I can take this one if you want.)

Reply:
Thanks Zane!